src/main/java/page/RegisterPage.java

```diff
@@ -61,77 +61,88 @@
     @FindBy(xpath = "//input[@value='Continue']")
     private WebElement button;
 
+
     public RegisterPage(WebDriver driver) {
         super(driver);
     }
 
 
-    public WebElement getFirstName() {
-        return firstName;
-    }
-
-    public WebElement getLastName() {
-        return lastName;
+    public RegisterPage fillFirstName(String userFirstName){
+        fillField(firstName, userFirstName);
+        return this;
```

Comment:
This method should return a page with a filled "name field". He will do it without pressing a button?